browser-extension/src/browserAction/menu.js

```diff
@@ -108,6 +108,12 @@ function openOptionsPage() {
   chrome.runtime.openOptionsPage();
 }
 
+function rateLaterHistory() {
+  browser.tabs.create({
```

Comment:
Chrome does not support the `browser` namespace, whereas Firefox supports `chrome` :upside_down_face: 
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API#browser_api_differences

```suggestion
  chrome.tabs.create({
```